src/Hl7.Fhir.Specification/Specification/Terminology/ValueSetExpander.cs

```diff
@@ -264,9 +289,27 @@ internal static ValueSet.ContainsComponent ToContainsComponent(this CodeSystem.C
 
             if (source.Concept.Any())
                 newContains.Contains.AddRange(
-                    source.Concept.Select(c => c.ToContainsComponent(system)));
+                    source.Concept.Select(c => c.ToContainsComponent(system, settings)));
 
             return newContains;
         }
+
+        private static List<ValueSet.DesignationComponent> ToValueSetDesignations(this List<CodeSystem.DesignationComponent> csDesignations)
```

Comment:
Throughout the api we normally don't prefix the parameters, variables, etc with their abbreviated types. It is not wrong, but it is more a certain style